jetty9/src/main/docker/Dockerfile

```diff
@@ -16,7 +16,11 @@ FROM ${docker.openjdk.image}
 
 # create jetty group and user
 RUN groupadd -r jetty \
- && useradd -r -g jetty jetty
+ && useradd -r -g jetty jetty \
+ && apt-get -q update \
+ && apt-get -y -q --no-install-recommends install sudo \
```

Comment:
Note that [Docker official best practices doc](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/user) recommends against installing sudo. I think we should consider alternatives.